server/routes/auth.js

```diff
@@ -158,14 +166,15 @@ router.patch('/users/:username', (req, res, next) => {
 });
 
 router.put('/users', (req, res, next) => {
-  Users.createUser(
+  req.services.userService.createUser(
+    Users,
     {
       username: req.body.username,
       password: req.body.password,
       host: req.body.host,
       port: req.body.port,
       socketPath: req.body.socketPath,
-      isAdmin: false,
+      isAdmin: req.body.isAdmin === '1',
```

Comment:
I'd prefer we transform this to a boolean client-side at https://github.com/jfurrow/flood/pull/718/files#diff-fcee3080ef44f99a921c5a3d9168680cR123